include/api_manager/utils/status.h

```diff
@@ -115,6 +120,11 @@ class Status final {
 
   // Error cause indicating the origin of the error.
   ErrorCause error_cause_;
+
+  // Detailed information of a grpc response error. If gRPC response trailers 
+  // have a "grpc-status-details-bin" header, the decoded value of that 
+  // header is stored here.
+  std::string grpc_status_details_;
```

Comment:
I don't think this is correct.  My understanding:  "grpc-status-detail-bin" is a proto binary of rpc:status which is error_code and error_message.  We need to:
* get the header value, base64 decode it
* call ParseFromString() to convert binary into proto
* assign the code and error_message